spec/requests/categories_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'CategoryController', type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get '/'
+      expect(response).to have_http_status(302)
+    end
+
+    it 'returns an error response for a wrong path' do
+      get '/failed_path'
+      expect(response).to have_http_status(404)
+    end
+
+    it 'should redirects to the sign in page for an unauthenticated user' do
+      get '/categories'
+      expect(response).to redirect_to(new_user_session_path)
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done, but one of the `test` failed on this `spec` file on my side, please fix it because all `tests` should be passed 🙏🏼 

![image](https://github.com/ivangonzalez224/Budget-Boss/assets/63915024/51ce2c45-4127-4dca-9356-b9a97125361f)

Reply:
Hello @mukhammadrizooff , thank you for the review. I made a mistake with the successful response code, it is 200
Changes done: all tests passed, fix the test for the successful response for the root path.

![test_passed](https://github.com/ivangonzalez224/Budget-Boss/assets/132683272/53d6de23-2bb2-4d7d-b026-f91643695038)